content/topics/unit-testing/tdd-horrors/_index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: TDD Horrors
+ready: true
+---
+
+These are all the most common TDD problems recruits tend to come across, read and make sure you didn't make any of these or any mistakes at all.
+
+- **Write tests**. Recruits in the past have done the mistake of not writing tests for TDD assignments, make sure you avoid this weird mistake. always write tests for your TDD projects.
+- Please make sure you understand .gitignore, please don't add your node_modules to git.
+- Name your files according to what is inside them
+- Avoid making tests depend on each other, either explicitly or implicitly. Dependencies among tests are a path to pain, expense, fragility, and complication.
```

Comment:
Can you simplify or elaborate in this? Newbies won't know what any of these words mean.